### ISSUE TITLE
browser chat not restarting conversation

### ISSUE THREAD

**Issue by aslu98** [facebookresearch/ParlAI#3033]
**Bug description**
When I use the browser chat service and push the "Restart Conversation" button it clears the chat but when I type a generic message like "Hello" the bot answers refers to the context of the previous conversation (ex. "What grade are you teaching?" as the conversation was about school and teaching).

**Reproduction steps**
Use browser chat mode (running browser_chat/run.py and browser_chat/client.py)

**Expected behavior**
I expected the bot to answer with a generic greeting form, not a specific question.

**Additional context**
I'm using a model trained from the blender90M model with bst and my own dataset.
I'm using the world in the "chatbot" folder.

Also, I tried to put `single_turn:True` in the config file because I want to reset the bot context every QA pair, but that did not work. Is this possible?

**Comment by klshuster** [comment]
Thanks for bringing this to our attention - a bug fix is in PR #3050, will comment on here when it lands!

**Comment by klshuster** [comment]
The fix has landed, please pull from master and try again - closing for now, but feel free to reopen if the issue persists.

**Comment by aslu98** [comment]
Thanks for the fix.
I pulled and tried the chat again with similar texts but the issue seems to still be there; when I push "Restart Conversation" button and say "Hello", I get an answer referring to the previous conversation.
I deleted some lines in the client script so I can restart the conversation with the string [DONE] as I do when using interactive.py in the shell, but I would not know how to have the "Restart Conversation" button doing the same.
